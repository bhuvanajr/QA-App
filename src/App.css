@import 'bootstrap/dist/css/bootstrap.css';
@import './styles/main.css';

:root {
  --qa-green: #354328;
}

#root {
  position: relative;
  padding-bottom: 6.25rem;
  min-height: 100%;
  font-family: Geomanist, Helvetica, Arial, sans-serif;
}

.application-container {
  text-align: left;
  background-color: #b1de87;
  min-height: 100%;
}

.container {
  width: 100%;
  padding-right: 8px;
  padding-left: 8px;
}

.title {
  display: block;
  font-weight: 600;
  width: 100%;
}

.subtitle {
  display: block;
  font-weight: 500;
  text-transform: none;
  width: 100%;
  font-size: 1.25rem;
  margin-top: 0px;
  margin-bottom: 0.5em;
}

.title-block {
  position: relative;
  max-width: 100%;
  margin-right: auto;
  margin-left: auto;
  background: none;
  text-align: center;
  padding-top: 30px;
  padding-bottom: 30px;
}

tr th.text-primary {
  color: #596f44 !important;
}

div .container-lg {
  margin-bottom: 4%;
}

main {
  padding-bottom: 30px;
}

.details {
  background-color: #ffffff;
}

input[type='checkbox'] {
  accent-color: var(--qa-green);
}
