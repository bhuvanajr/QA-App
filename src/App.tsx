import React from 'react';
import { ContainerFooter, ContainerHeader, QAOverview } from './components';

const App: React.FC = () => {
  return (
    <div className="application-container">
      <header>
        <ContainerHeader />
      </header>
      <main>
        <QAOverview />
      </main>
      <footer>
        <ContainerFooter />
      </footer>
    </div>
  );
};

export default App;
