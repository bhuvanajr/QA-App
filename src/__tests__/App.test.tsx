import { render } from '@testing-library/react';
import React from 'react';
import App from '../App';

test('should render correctly', () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
