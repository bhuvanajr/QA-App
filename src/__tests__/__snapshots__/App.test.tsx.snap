// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  <div
    class="application-container"
  >
    <header>
      <div>
        ContainerHeader
      </div>
    </header>
    <main
      class="container"
    >
      <div>
        QAOverview
      </div>
    </main>
    <footer>
      <div>
        ContainerFooter
      </div>
    </footer>
  </div>
</div>
`;
